src/components/CustomerQuota/ItemsSelection/ItemStepper.tsx

```diff
@@ -14,6 +14,8 @@ import { ProductContext } from "../../../context/products";
 import { sharedStyles } from "./sharedStyles";
 
 export const ItemStepper: FunctionComponent<{
```

Comment:
How are you thinking of handling the addons for this component?

Reply:
will remove them for now since the props are not compulsory.

For the handling, thinking of making them look similar to ItemCheckbox's one on another PR once this one is approved!